onmt/modules/JoinReplicateTable.lua

```diff
@@ -0,0 +1,140 @@
+local JoinReplicateTable, parent = torch.class('onmt.JoinReplicateTable', 'nn.Module')
```

Comment:
Does this file come from somewhere else? To be consistent with the rest of the project, indentation should be 2 spaces and variable names should be in camelCase.